### DISCUSSION TITLE
✨ Feature Request: Add script for building scss/sass modules

### DISCUSSION THREAD

**Discussion by chtushar** [radix-ui/colors#58] [Ideas]
I've been using `@radix-ui/colors` for a while now. Recently, working on a small project I noticed there isn't any scss/sass modules exported from the library. I understand that we can directly use css variables in sass files. But, it is always better if we are able to use `$blue7` instead of `var(--blue7)`. 

**I would love to work on this feature creating a PR for the same.** 

Thank-you 😊

**Comment by colmtuite** [comment]
Any thoughts on this @vladmoroz @peduarte?
